test-common/src/main/daml/model/Test.daml

```diff
@@ -438,31 +436,6 @@ template DecimalRounding
   where
     signatory party
 
-{-
-testDivulgence : Scenario ()
-testDivulgence = test do
-  alice <- getParty "alice"
-  bob <- getParty "bob"
-  div1Cid <- submit alice (create Divulgence1 with div1Party = alice)
-  div2Cid <- submit bob (create Divulgence2 with div2Signatory = bob, div2Fetcher = alice)
-  submit alice (exercise div2Cid Divulgence2Fetch with div1ToFetch = div1Cid)
--}
-
-testScenario : Scenario ()
```

Comment:
as far as I could tell, nothing evaluates this, `testDivulgenceVisibility` nor `testDivulgenceSuccess`

Reply:
I'm not sure about _everything_ but quite a few of the templates are used, e.g. https://github.com/digital-asset/daml/blob/5784a54485712499e714a0c37b943d97cd88eaf2/ledger-test-tool/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_8/ActiveContractsServiceIT.scala#L21-L31 https://github.com/digital-asset/daml/blob/5784a54485712499e714a0c37b943d97cd88eaf2/ledger-test-tool/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_8/CommandServiceIT.scala#L19-L22 https://github.com/digital-asset/daml/blob/5784a54485712499e714a0c37b943d97cd88eaf2/ledger-test-tool/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_8/WitnessesIT.scala#L7-L9

Reply:
I had a look too, its seems to be used in quite a lot of places - i think finding unused templates in there would be difficult (unless you just delete and run all tests lol)